guides/common/modules/con_provisioning-methods-in-project.adoc

```diff
@@ -3,29 +3,33 @@
 [id="Provisioning-Methods-in-{ProjectNameID}_{context}"]
 = Provisioning methods in {ProjectName}
 
-With {ProjectName}, you can provision hosts by using the following methods.
+With {ProjectName}, you can provision hosts by using the following provisioning methods.
 
-Bare-metal hosts::
-{Project} provisions bare-metal hosts primarily by using PXE boot and MAC address identification.
-When provisioning bare-metal hosts with {Project}, you can do the following:
+Network based::
+You can use this method with hosts that provide PXE booting or UEFI HTTP booting.
```

Comment:
```suggestion
You can use this method with hosts that can utilize PXE booting or UEFI HTTP booting.
```
or
```suggestion
You can use this method with hosts that provide PXE booting or UEFI HTTP booting capabilities.
```

Otherwise it might sound like the hosts are the ones that offer PXE/HTTP boot services (which they don't)